augur/export_v2.py

```diff
@@ -690,6 +690,7 @@ def register_arguments_v2(subparsers):
         title="OPTIONAL SETTINGS"
     )
     optional_settings.add_argument('--minify-json', action="store_true", help="export JSONs without indentation or line returns")
+    optional_settings.add_argument('--include-root-sequence', action="store_true", help="Export an additional JSON containing the root sequence used to identify mutations. The filename will follow the pattern of <ROOT>_root-sequence.json for a main auspice JSON of <ROOT>.json")
```

Comment:
I think the example of `<ROOT>_root-sequence.json` is a little confusing because the word "root" is used in two different ways: first as the placeholder referring to the main output file's basename/stem and second as in the root of a tree. An alternative placeholder would be great. Maybe…

> The filename will follow the pattern of `<OUTPUT>_root-sequence.json` for a main auspice JSON of `<OUTPUT>.json`

or 

> The filename will follow the pattern of `<NAME>_root-sequence.json` for a main auspice JSON of `<NAME>.json`